src/towncrier/check.py

```diff
@@ -81,20 +71,16 @@ def __main(comparewith, directory, config):
     encoding = getattr(sys.stdout, "encoding", "utf8")
```

Comment:
This doesn’t do what the comment above says it does, right? If `sys.stdout.encoding is None` then `encoding` will be `None`.

Reply:
Yeah it absolutely doesn't. I'll remove the construct, because we don't do Python 2 anymore.